src/views/MainView/MainView.modules.css

```diff
@@ -0,0 +1,3 @@
+.mainView   {
+    padding: 20px
```

Comment:
Looks like the spacing on this file is a little off in comparison to the other files